### PR TITLE
Updated h3 core to 3.4.0 and updated tests

### PR DESCRIPTION
**Pull request by squiidz**
Fixed the issue #7. This pull request doesn't add the newest features from the core library it only update the ones already provided in the bindings

### CODE REVIEW

**Comment by bsr203** [comment]
@squiidz thanks for the update. quickly scrolling through, I only see one go test file is modified. Have you only had to run `update-h3.sh`. I wonder is that easy for a future update if I too only use a subset of the API and core didn't have any breaking API change.

**Comment by squiidz** [comment]
@bsr203 yes, I changed the version in H3_VERSION and used ```update-h3.sh``` and updated the test file.

**Comment by nrabinowitz** [comment]
Thanks for your contribution. Please note that @joegilley, the primary maintainer of `h3-go`, is out and won't be able to respond to this for several more weeks. Please hold tight until then - thanks!

File: h3_test.go [hunk 1]
```diff
@@ -84,17 +86,24 @@ var (
...
-		{Latitude: 67.160561948, Longitude: 191.845198829},
-		{Latitude: 67.234563187, Longitude: 191.713897218},
+		{Latitude: 67.224749856, Longitude: -168.523006585},
+		{Latitude: 67.140938355, Longitude: -168.626914333},
+		{Latitude: 67.067252558, Longitude: -168.494913285},
+		{Latitude: 67.077062918, Longitude: -168.259695931},
+		{Latitude: 67.160561948, Longitude: -168.154801171},
+		{Latitude: 67.234563187, Longitude: -168.286102782},
+
+		// {Latitude: 67.224749856, Longitude: 191.476993415},
```
Comment (joegilley) [inline h3_test.go]: can you remove the commented out entries?  no longer necessary thx to you :D

File: h3_test.go [hunk 2]
```diff
@@ -106,6 +115,7 @@ func TestFromGeo(t *testing.T) {
 		Latitude:  67.194013596,
 		Longitude: 191.598258018,
 	}, 5)
+	t.Logf("Index:%x\n", h)
```
Comment (joegilley) [inline h3_test.go]: bit of a nit, but can you remove this line, or alternatively, rephrase it inside `assert.Equal`'s message field, perhaps like:
```
assert.Equal(t, validH3Index, h, 
  "expected %x but got %x", 
  validH3Index, h)
```

**Comment by squiidz** [comment]
@joegilley Cleanup done :) For the CLA, i've tried to sign it many times but it didn't seems to update the status in the pull request.

**Comment by squiidz** [comment]
@joegilley up

**Pull request merged**